build-logic/src/main/kotlin/dokkabuild/DevMavenPublishExtension.kt

```diff
@@ -42,14 +49,13 @@ abstract class DevMavenPublishExtension(
 
         task.dependsOn(devMavenRepositories)
 
-        if (task is JavaForkOptions) {
-            task.doFirst("devMavenRepositories systemProperty") {
-                // workaround https://github.com/gradle/gradle/issues/24267
-                task.systemProperty(
-                    "devMavenRepositories",
-                    devMavenRepositories.joinToString(",") { it.canonicalFile.invariantSeparatorsPath }
-                )
-            }
+        if (task is Test) {
```

Comment:
BTW, is there a case when we need it for non-`Test` task? May be just make `configureTask` accept `Test`?
I mean, it has a rather specific usage pattern.

Reply:
Good idea, done.